ydb/tests/functional/tpc/canondata/test_init.TestClickbenchInit.test_s1_column/s1_column

```diff
@@ -0,0 +1,121 @@
+Init tables ...
+--!syntax_v1
+
+
+CREATE  TABLE `/Root/db/Root/db/clickbench/s1`
+(
```

Comment:
Parenthesis after CREATE TABLE <name> is on another line. There are examples where it is on the same line. I prefer seeing it on the same line. At least let it be the same behaviour for all places